TimetableApp/TimetableApp.iOS/TimetableApp.iOS.csproj

```diff
@@ -115,6 +115,9 @@
     <BundleResource Include="Resources\Fonts\uno-fluentui-assets.ttf" />
   </ItemGroup>
   <ItemGroup>
+    <PackageReference Include="ToastNotification.Uno">
+      <Version>0.0.0-preview-18-835992017</Version>
```

Comment:
The iOS version is not released yet, but well... it should be here, so I'd say that's fine?

Reply:
you said the ToastNotification.uno is for all other platforms :))

Reply:
Yeah it's alright.